modules/navier_stokes/test/tests/finite_volume/ins/turbulence/channel/segregated/channel_ERCOFTAC.i

```diff
@@ -123,7 +141,7 @@ wall_treatment = 'eq_newton' # Options: eq_newton, eq_incremental, eq_linearized
     variable = vel_x
     mu = 'mu_t'
     momentum_component = 'x'
-    complete_expansion = true
+    complete_expansion = no
```

Comment:
the deviatoric part should be fixed now, at least for laminar cases.

Reply:
Yes, but we don't need it for this a channel case. There is no need to test it here